includes/Admin/SetupWizard.php

```diff
@@ -270,8 +270,14 @@ public function setup_wizard_header() {
             <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
             <title><?php esc_html_e( 'Dokan &rsaquo; Setup Wizard', 'dokan-lite' ); ?></title>
             <?php wp_print_scripts(); ?>
-            <?php do_action( 'admin_print_styles' ); ?>
-            <?php do_action( 'admin_head' ); ?>
+            <?php
+            wp_enqueue_emoji_styles();
+            do_action( 'admin_print_styles' );
+            ?>
+            <?php
+            wp_enqueue_admin_bar_header_styles();
+            do_action( 'admin_head' );
+            ?>
             <?php do_action( 'dokan_setup_wizard_styles' ); ?>
```

Comment:
Pls put all *PHP* codeblock inside a single `php` tag.

Reply:
Working on it bhai